src/rgw/CMakeLists.txt

```diff
@@ -299,10 +299,12 @@ target_link_libraries(rgw_common
     cls_version_client
     librados
     libneorados
+    keyring
     rt
     ICU::uc
     OATH::OATH
     dmclock::dmclock
+    keyutils
```

Comment:
cmake nit: the `keyring` target should itself carry this `keyutils` dependency so rgw doesn't need to track it
```cmake
target_link_libraries(keyring PUBLIC keyutils)
```